### ISSUE TITLE
Admin login screen not showing up, logs reveal little

### ISSUE THREAD

**Issue by carlvlewis** [opensavannah/bizport#6]
So I took a look at the Documentation to figure out the CMS. It seems that whenever I go to `admin/users` or even just `/admin/` it throws back the following error message:

![](http://cvlassets.s3.amazonaws.com/Screen%20Shot%202017-12-30%20at%206.25.02%20AM.png)

I've tried redeploying from @Bentley912's branch. It deploys fine, but still no CMS abilities. Patrick, have you gotten this to work yet?

**Comment by Bentley912** [comment]
No. I saw that. I don't know why either. I checked the logs. I will go over
the deploy docs again and make sure I didn't miss a step. Maybe I didn't
migrate everything over to Heroku properly?
